codespell_lib/data/dictionary.txt

```diff
@@ -2592,17 +2597,19 @@ asuming->assuming
 asumption->assumption
 asure->assure
 aswell->as well
+asychronize->asynchronize
+asychronized->asynchronized
 asychronous->asynchronous
 asychronously->asynchronously
 asycn->async
 asycronous->asynchronous
 asymetic->asymmetric
-asymetri->assymetry
+asymetri->asymmetry, asymmetric,
 asymetric->asymmetric
 asymetrical->asymmetrical
 asymetricaly->asymmetrically
 asymmeric->asymmetric
-asymmetri->asymmetric
+asymmetri->asymmetric, assymmetry,
```

Comment:
It looks weird that we have
```
asymetri->asymmetry, asymmetric,
asymmetri->asymmetric, assymmetry,
```
1) very similar typos with suggestions in a different order
2) two differently-spelled suggestions for asymmetry ??

Reply:
fixed, fixed and added as typo

Reply:
Well spotted! assymmetry and it's equivalent extras needs adding as a typo too!

That's possibly historic. Although I'd argue asymmetri is a dropped letter away from asymmetric, whereas asymetri is a lot of mistyping but phonetically a good attempt at asymmetry. Also generally asymmetry is slightly more popular, so should probably be suggested first if there's not a clear winner/obvious typo I'd suggest.

Reply:
@sebweb3r , if we're going to standardise them, can we do it based on the typo not the phonetics.

Reply:
you mean reverse the order? then I've misunderstood your comment :-D

Reply:
I'm arguing for not standardising, i.e. leaving it as it was. If @lurch feels strongly then put assymetric first in both, but there will be loads more similarly random entries across the dictionary.

Reply:
ok, if he doesn't care, I leave it as it is right now (asymmetry first). I remain indifferent.

Reply:
I'm not bothered :+1:

Reply:
Sorry, to clarify I'd like asymmetri to have asymmetric first (given it's just one dropped letter). I'm less bothered about what is first for the asymetri typo.

Reply:
ok. then I put asymmetric first for both typos

Reply:
Thanks